### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by venikunche**

### CODE REVIEW

**Comment by khandelwal** [comment]
Hello! If you update this pull request (and fix the conflicts as well) - I think the tests should now pass. We pushed out a pull request that fixes the breaking tests.

File: ekip/config/settings/base.py [hunk 1]
```diff
@@ -90,6 +92,11 @@
 # Internationalization
 # https://docs.djangoproject.com/en/1.7/topics/i18n/
 
+LANGUAGES = (
+    ('en-us', _('English')),
+    ('es-us', _('Spanish')),
```
Comment (khandelwal) [inline ekip/config/settings/base.py:97]: Should the label here actually say 'Spanish' in Spanish?

File: ekip/everykid/static/css/voucher.css [hunk 2]
```diff
@@ -31,7 +31,7 @@ html, body {
 
 .voucher-container, .voucher-instructions {
   width: 610px;
-  height: 842px;
+  height: 880px;
```
Comment (khandelwal) [inline ekip/everykid/static/css/voucher.css:34]: Why did this change based on a language change?

File: ekip/everykid/templates/get-your-pass/educator_passes.html [hunk 3]
```diff
@@ -41,30 +41,29 @@
...
-      <p>Activity 2: <a href="{% static "activities/scholastic-two.pdf" %}" target="_blank">Environmental Stewardship</a> (PDF). This lesson shows students how to take care of lands and waters.</p>
-      <p>Activity 3: <a href="{% static "activities/scholastic-three.pdf" %}" target="_blank">Our Nation’s Native Peoples</a> (PDF). This lesson teaches students about the people who lived on this land before it was called the United States.</p>
-      <p>Activity 4: <a href="{% static "activities/scholastic-four.pdf" %}" target="_blank">Citizen Science</a> (PDF). This lesson helps kids learn about the difference between weather and climate.</p>
+      <h3>{% trans "Download" %}<br/> {% trans "Activity Guides" %}</h3>
+      <p>{% trans "Activity" %} 1: <a href="{% if LANGUAGE_CODE == 'es-us' %}{% static "activities/es_us/scholastic-one.pdf" %}{% else %}{% static "activities/scholastic-one.pdf" %}{% endif %}" target="_blank">{% trans "Exploring Federal Lands and Waters" %}</a> (PDF). {% trans "This lesson teaches students why our country protects lands and waters." %}</p>
```
Comment (khandelwal) [inline ekip/everykid/templates/get-your-pass/educator_passes.html]: In the 'URL' for the PDF, I might use the LANGUAGE_CODE parameter, instead of hard-coding 'es-us'

File: ekip/everykid/templates/about.html [hunk 4]
```diff
@@ -141,23 +123,23 @@
...
+      <p>{% trans "Here's a" %} <a href="{% static "media/tool-kit-2.zip" %}">{% trans "set of resources" %}</a> (ZIP file, 15.4 MB) {% trans "to help local officials, youth leaders and others plan an outdoor outing with 4th graders." %}</p>
+      <p>{% trans "In partnership with the Every Kid in a Park Program, Transforming Youth Outdoors" %} <a href="https://tyo.pathgather.com/l/login" target="_blank">{% trans "has created two step-by-step learning paths" %}</a> {% trans "that give classroom teachers and outdoor educators tools to learn more about the Every Kid program and plan a successful field trip. These resources provide information about federal lands and waters, tools to find nearby locations, and information on planning and executing an Every Kid in a Park field trip." %}</p>
```
Comment (khandelwal) [inline ekip/everykid/templates/about.html:136]: I would recommend wrapping long text lines such as these, so that it's a bit easier to read.

File: ekip/everykid/templates/index.html [hunk 5]
```diff
@@ -29,23 +30,35 @@
...
         </li>
         <li>
-          <a href="{% url 'plan_your_trip' %}">Plan a Trip</a>
+          <a href="{% url 'plan_your_trip' %}">{% trans "Plan a Trip" %}</a>
         </li>
         <li>
-          <a href="{% url 'parents' %}">Parents</a>
-        </li>
+          <a href="{% url 'parents' %}">{% trans "Parents" %}</a>
+        </li>
```
Comment (khandelwal) [inline ekip/everykid/templates/index.html:42]: This form seems to be duplicated in multiple locations. With template inheritance, I think you shouldn't need to to this. At the least, you should be able to type it out once, and then just do an include.

File: ekip/everykid/templates/nav.html [hunk 6]
```diff
@@ -1,15 +1,28 @@
...
-        <a href="{% url 'parents' %}">Parents</a>
+        <a href="{% url 'parents' %}">{% trans "Parents" %}</a>
       </li>
       <li class="{% active request 'educator' %}">
-        <a href="{% url 'educator_passes' %}">Educators</a>
+        <a href="{% url 'educator_passes' %}">{% trans "Educators" %}</a>
       </li>
       <li class="{% active request 'about' %}">
-        <a href="{% url 'about_ekip' %}">About</a>
+        <a href="{% url 'about_ekip' %}">{% trans "About" %}</a>
```
Comment (khandelwal) [inline ekip/everykid/templates/nav.html:13]: Repeated form.

File: requirements.txt [hunk 7]
```diff
@@ -1,13 +1,13 @@
 # Django, using PostgreSQL
-Django==1.9.6
+Django==1.9.10
```
Comment (khandelwal) [inline requirements.txt:2]: This should not have changed from Master. Is this pull request updated?

File: ekip/everykid/urls.py [hunk 8]
```diff
@@ -30,36 +28,39 @@
     url(
         r'get-your-pass/educator', EducatorFormPreview(EducatorForm),
         name="educator_passes"),
-    url(r'get-your-pass/', cache_page(60*60)(TemplateView.as_view(
-        template_name="get-your-pass/index.html")), name="get_your_pass"),
+    url(r'get-your-pass/', TemplateView.as_view(
```
Comment (khandelwal) [inline ekip/everykid/urls.py]: Removing all this caching is making me nervous. Before this hits production, I think you need to look at how to make the CSRF stuff work with caching. I believe there's some documentation on the Django website that talks to this.

File: ekip/redemption/views.py [hunk 9]
```diff
@@ -202,15 +202,17 @@ def tables(request):
...
 def statistics(request):
 
-    educator_tickets = Educator.objects.all().aggregate(
+    # Filter out large - possibly abandoned requests.
+    educator_tickets_cleaned = Educator.objects.filter(num_students__lte=50).aggregate(
         Sum('num_students'))['num_students__sum']
 
-    if not educator_tickets:
-        educator_tickets = 0
+    if not educator_tickets_cleaned:
```
Comment (khandelwal) [inline ekip/redemption/views.py:209]: How is this change related to the Spanish translation?

File: ekip/config/settings/base.py (hunk 1 above)
Comment (venikunche) [reply ekip/config/settings/base.py:97]: No. I've put all of the Spanish translations in the .po files.

File: ekip/everykid/static/css/voucher.css (hunk 2 above)
Comment (venikunche) [reply ekip/everykid/static/css/voucher.css:34]: The Spanish text takes more space. With the 842px height, the text was running out of the border like this:
screen shot 2016-11-30 at 10 44 37 am

File: ekip/everykid/templates/get-your-pass/educator_passes.html (hunk 3 above)
Comment (venikunche) [reply ekip/everykid/templates/get-your-pass/educator_passes.html]: That makes sense. I'll make the change.

File: ekip/everykid/templates/about.html (hunk 4 above)
Comment (venikunche) [reply ekip/everykid/templates/about.html:136]: Unfortunately, when using {% trans "text" %}, all of the text had to be in one line. The translations I was given were by paragraphs. Since the structure of a sentence in English and Spanish may be different. I wasn't sure on how to break them up.

File: ekip/everykid/templates/index.html (hunk 5 above)
Comment (venikunche) [reply ekip/everykid/templates/index.html:42]: Agreed. For now, I focused on just translating the site and tried not to change the original site unless necessary.

Comment (venikunche) [reply ekip/everykid/templates/index.html:42]: I misunderstood your comment earlier. Yes, it makes sense to type it out once and then just do an include.

File: requirements.txt (hunk 7 above)
Comment (venikunche) [reply requirements.txt:2]: The pull request is updated. It looks like it is the right version. The [master](https://github.com/18F/ekip-api/blob/c1b20b40a3a10e9876e68be7537ed2709ff945b9/requirements.txt) also has 1.9.10.

File: ekip/redemption/views.py (hunk 9 above)
Comment (venikunche) [reply ekip/redemption/views.py:209]: Not sure on what's going on here. I did not make any changes to this file. I looked at [our repository](https://github.com/USGS-WiM/ekip-api/commits/spanish-translation/ekip/redemption/views.py) and it hasn't been modified. I compared it with what's in master and it's the same. I am not sure on why GitHub is showing it as though there is a difference.

File: ekip/everykid/urls.py (hunk 8 above)
Comment (venikunche) [reply ekip/everykid/urls.py]: Sounds good. I'll take a look.
